### ISSUE TITLE
Support Air-Gapped environments

### ISSUE THREAD

**Issue by yahavi** [jfrog/frogbot#157]
**Is your feature request related to a problem? Please describe.**
Frogbot does not support Air-Gapped environments.

**Describe the solution you'd like to see**
Allow downloading Frogbot, extractors, and project dependencies from Artifactory.

**Describe alternatives you've considered**
N/A

**Additional context**
This is just a suggestion.
Let's split the solutions into 2 parts:
1. Download Frogbot and extractors from Artifactory - 
  A. Create a virtual repository in releases.jfrog.io containing [frogbot](https://releases.jfrog.io/artifactory/frogbot) and [oss-release-local](https://releases.jfrog.io/artifactory/oss-release-local)
  B. Create a new script `getFrogbot.sh` and upload it to the frogbot repository. The script should support an environment variable `FROGBOT_REMOTE_REPO` and the regular frogbot connection details variables. Suggestion - users can put this script in an accessible place. It is better if this script will have a version.
  C. Add support for `FROGBOT_REMOTE_REPO` env var in frogbot to download Maven and Gradle extractors.
2. Download dependencies from Artifactory - support the same project yaml files as the CLI does. Ignore the server ID and read only the resolution server. Additionally, for simplicity, we can allow users who like to provide `FROGBOT_RESOLUTION_REPO` variable to download dependencies from Artifactory.

**Comment by EvertonSA** [comment]
+1

**Comment by eyalbe4** [comment]
@omerzi is working on adding this functionality to Forgbot these days. We're planning to introduce this functionality in the very near future.

**Comment by omerzi** [comment]
Hi @EvertonSA, @Zoan112, @codanael, we just released Frogbot [v2.6.0](https://github.com/jfrog/frogbot/releases/tag/v2.6.0) with air-gapped environments support.

**Comment by eyalbe4** [comment]
Adding to @omerzi  message above, Forgbot can now be configured to access Artifactory only. The Frogbot executable, as well the scanned project depedencies, can be downloaded through Artifactory. This requires creating the relevant repositories in Artifactory, and adding their names as variables in the Forgbot workflows. 
See the two variables below. This information is included in the Forgbot templates in the documentation. 
```
# [Optional]
# If the machine that runs Frogbot has no access to the internet, set the name of a remote repository 
# in Artifactory, which proxies https://releases.jfrog.io/artifactory
# The 'frogbot' executable and other tools it needs will be downloaded through this repository.
# JF_RELEASES_REPO: ""

# [Optional]
# Frogbot will download the project dependencies if they're not cached locally. To download the
# dependencies from a virtual repository in Artifactory, set the name of the repository. There's no
# need to set this value, if it is set in the frogbot-config.yml file.
# JF_DEPS_REPO: ""
```

If your repository uses a `frogbot-config.yml` file, you can define the JF_DEPS_REPO value as the `repository` value inside the file.
